electrumpersonalserver/server/common.py

```diff
@@ -132,11 +133,11 @@ def send_reply_fun(reply):
                         line = recv_buffer[:lb].rstrip()
                         recv_buffer = recv_buffer[lb + 1:]
                         lb = recv_buffer.find(b'\n')
-                        line = line.decode("utf-8")
-                        logger.debug("=> " + line)
                         try:
+                            line = line.decode("utf-8")
+                            logger.debug("=> " + line)
```

Comment:
Move this line outside the try/except block, to limit the scope of the exceptions to as little code as possible.

Reply:
sorry, I missed that. The debug line is now after the try/except block.